### PR TITLE
Add HTTP Agent deployment

### PR DESCRIPTION
**Pull request by danielapetito**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
Add HTTP Agent deployment

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:
dojot/dojot#2246

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -811,6 +811,57 @@ services:
...
+      HTTP_AGENT_HTTPS_CA: '/certs/ca.crt'
+      HTTP_AGENT_SECURITY_CRL: '/certs/ca.crl'
+      HTTP_AGENT_SECURITY_CERT_DIRECTORY: '/certs'
+      HTTP_AGENT_LOG_CONSOLE_LEVEL: 'info'
+    volumes:
+      - http-agent-cert-side-volume:/certs
+    logging:
+      driver: json-file
+      options:
+        max-size: 100m
```
Comment (marianoleonardo) [inline docker-compose.yml:862]: I think you should change this to:
```
        max-size: 20m
        max-file: '5'
```
This way, the log will be rotative

File: docker-compose.yml [hunk 2]
```diff
@@ -811,6 +811,57 @@ services:
...
+      CERT_SC_CERTS_HOSTNAMES: '["http-agent", "localhost"]'
+      HOSTNAME: http-agent
+      CERT_SC_CERTS_FILES_BASEPATH: /certs
+      CERT_SC_CERTS_FILES_CA: ca.crt
+      CERT_SC_CERTS_FILES_CERT: http-agent.crt
+      CERT_SC_CERTS_FILES_KEY: http-agent.key
+      CERT_SC_CERTS_FILES_CRL: ca.crl
+      CERT_SC_CRON_CRL_TIME: '0 */1 * * * *'
+    volumes:
+      - http-agent-cert-side-volume:/certs
```
Comment (marianoleonardo) [inline docker-compose.yml:830]: Just an aesthetic comment... I think the volume name could follow the "dojot pattern", so for this you need to change `http-agent-cert-side-volume` to `http-agent-cert-sidecar-volume`.

File: docker-compose.yml [hunk 3]
```diff
@@ -1101,6 +1152,7 @@ volumes:
   mosca-redis-volume:
   influxdb-volume:
   acl-redis-volume:
+  http-agent-cert-side-volume:
```
Comment (marianoleonardo) [inline docker-compose.yml:1155]: Just an aesthetic comment... I think the volume name could follow the "dojot pattern", so for this you need to change `http-agent-cert-side-volume` to `http-agent-cert-sidecar-volume`.

File: docker-compose.yml [hunk 4]
```diff
@@ -811,6 +811,57 @@ services:
...
+    restart: always
+    environment:
+      HTTP_AGENT_HTTPS_CERT: '/certs/http-agent.crt'
+      HTTP_AGENT_HTTPS_KEY: '/certs/http-agent.key'
+      HTTP_AGENT_HTTPS_CA: '/certs/ca.crt'
+      HTTP_AGENT_SECURITY_CRL: '/certs/ca.crl'
+      HTTP_AGENT_SECURITY_CERT_DIRECTORY: '/certs'
+      HTTP_AGENT_LOG_CONSOLE_LEVEL: 'info'
+    volumes:
+      - http-agent-cert-side-volume:/certs
```
Comment (marianoleonardo) [inline docker-compose.yml:858]: Is this right? The same volume was mapped for `http-agent-cert-sidecar`.
If is it right you should apply here the same aesthetic comment that i mentioned before.

**Review by marianoleonardo — CHANGES_REQUESTED** [review]
I left some comments.

File: docker-compose.yml (hunk 1 above)
Comment (danielapetito) [reply docker-compose.yml:862]: ok

**Review by marianoleonardo — APPROVED** [review]
Great Job!!! 👏🏼👏🏼👏🏼

**Review by gabrielifg — APPROVED** [review]
Nice!!!

**Pull request merged**